### DISCUSSION TITLE
CRITICAL Vulnerability found in non-os package type (npm) - /opt/code-server/lib/vscode/extensions/handlebars/package.json

### DISCUSSION THREAD

**Discussion by graczhual** [coder/code-server#6911] [Q&A]
https://www.npmjs.com/package/handlebars
Is this the built-in extension in vscode? How can we upgrade it? THX

**Comment by code-asher** [comment]
This is a false positive, so it can be ignored.  The vulnerability is in the handlebars package, not in the built-in extension.  They just happened to be named the same.

Related: https://github.com/coder/code-server/issues/6332
